examples/main.go

```diff
@@ -71,14 +72,14 @@ func main() {
 
 	switch r := resp.(type) {
 	case *momento.CacheGetHit:
-		log.Printf("Lookup resulted in cahce HIT. value=%s\n", r.ValueString())
+		mLogger.Info("Lookup resulted in cache HIT", "key", key, "value", r.ValueString())
 	case *momento.CacheGetMiss:
-		log.Printf("Look up did not find a value key=%s", key)
+		mLogger.Info("Look up did not find a value", "key", key)
 	}
 
 	// Permanently delete the cache
 	if err = client.DeleteCache(ctx, &momento.DeleteCacheRequest{CacheName: cacheName}); err != nil {
 		panic(err)
 	}
-	log.Printf("Cache named %s is deleted\n", cacheName)
+	mLogger.Info(fmt.Sprintf("Cache named %s is deleted\n", cacheName))
```

Comment:
nit we can change in next pr if we want but would normally write log message like this:
```suggestion
	mLogger.Info("Cache successfully deleted", "name", cacheName)
```